task/reindex_test.go

```diff
@@ -818,3 +818,78 @@ func TestAddTopicWithParents(t *testing.T) {
 		})
 	})
 }
+
+func TestMigratedContentFiltering(t *testing.T) {
+	Convey("Given a extractedChan channel and a transformedChan channel", t, func() {
+		errChan := make(chan error, 1)
+		extractedChan := make(chan Document, 1)
+		transformedChan := make(chan Document, 1)
+		tracker := &Tracker{}
+
+		Convey("When a non-editorial document with a migration link is sent to the extractedChan and consumed by transformZebedeeDoc", func() {
+			extractedChan <- Document{
+				ID:  "doc1",
+				URI: "economy/migrated-content",
+				Body: []byte(`{
+					"type": "static_landing_page",
+					"description": {
+						"title": "A static landing page title",
+						"migrationLink": "some-new-location"
+					}
+				}`),
+			}
+			close(extractedChan)
+
+			transformZebedeeDoc(ctx, tracker, errChan, extractedChan, transformedChan, nil)
+
+			Convey("Then the document is filtered out and not sent to the transformed channel", func() {
+				So(len(transformedChan), ShouldEqual, 0)
```

Comment:
There's a built in length check that you can use or better still there's a `ShouldBeEmpty`

```suggestion
				So(transformedChan, ShouldHaveLength, 0)
```

Reply:
Changed